Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -57,18 +57,18 @@ public protocol NavigationService: CLLocationManagerDelegate, RouterDataSource,
     var eventsManager: NavigationEventsManager { get }
     
     /**
-     The route along which the user is expected to travel, plus its index in the `RouteResponse`, if applicable.
+     The route along which the user is expected to travel.
 
      If you want to update the route, use `Router.updateRoute(with:routeOptions:)` method from `router`.
      */
-    var indexedRoute: IndexedRoute { get }
-
+    var route: Route? { get }
+    
     /**
-     The route along which the user is expected to travel.
-
+     The `RouteResponse` object containing active route, plus its index in this `RouteResponse`, if applicable.
+     
```

Comment:
nit: sneaky empty spaces :) I would love swift-format integrated so that we will never see such things ever again.